practice/src/css/sections/destination.css

```diff
@@ -0,0 +1,143 @@
+.wrapper-destination {
+  padding-bottom: 100px;
+}
+
+.destination-heading {
+  text-align: center;
+}
+
+.destination-card-item-image {
+  width: 100%;
+  height: auto;
+  margin-bottom: 20px;
+  border-radius: 24px;
+}
+
+.destination-type-trip {
+  font-weight: var(--font-weight-light);
+  font-size: var(--font-size-xxsmall);
+  line-height: var(--small-line-height);
+  font-style: italic;
+  color: var(--dark-text-color);
+}
+
+.destination-title {
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-medium);
+  line-height: var(--secondary-line-height);
+  color: var(--dark-text-color);
+}
+
+.destination-substact {
+  display: flex;
+  gap: 4px;
+}
+
+.btn-destination {
+  font-size: var(--font-size-small);
+  padding: 8px 24px;
+  border-radius: 12px;
+  background-color: var(--tertiary-background-color);
+}
+
+.booking-info {
+  display: flex;
+  gap: 2px;
+}
+
+.destination-book {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 20px;
+}
+
+.destination-address {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.destination-card-item {
+  padding-bottom: 20px;
+  position: relative;
+}
+
+.destination-rate {
+  border-radius: 50px;
+  background-color: rgba(255, 255, 255, 0.1);
+  padding: 4px 12px;
+  position: absolute;
+  top: 16px;
+  right: 20px;
+}
+
+.destination-rate-number {
+  color: var(--light-text-color);
+}
+
+.destination-description {
+  padding-bottom: 50px;
+}
+
+.destination-button {
+  padding-top: 50px;
+}
+
+.destination-card-item-image {
```

Comment:
Why do you split style for destination-card-item-image

```
.destination-card-item-image {
  width: 100%;
  height: auto;
  margin-bottom: 20px;
  border-radius: 24px;
}
```

Reply:
Because I added the raw image, not the design image on figma